gripper_node:
  ros__parameters:
    on_timeout: 100
